lib/Runtime/Library/TypedArray.cpp

```diff
@@ -143,7 +143,11 @@ namespace Js
             {
                 if (JavascriptOperators::IsObject(firstArgument))
                 {
-                    if (JavascriptOperators::IsIterable(RecyclableObject::FromVar(firstArgument), scriptContext))
+                    Var iterator = JavascriptOperators::GetProperty(RecyclableObject::FromVar(firstArgument), PropertyIds::_symbolIterator, scriptContext);
+
+                    if (!JavascriptOperators::IsUndefinedObject(iterator) &&
```

Comment:
Oh I see, `CreateNewInstanceFromIterableObj` doesn't handle the case where it is undefined.

Okay this is fine.